tests/baselines/reference/enumBasics.js

```diff
@@ -145,7 +145,7 @@ var E8;
 var E9;
 (function (E9) {
     E9[E9["A"] = 0] = "A";
-    E9[E9["B"] = 0] = "B";
+    E9["B"] = E9.A;

```

Comment:
Oops missing file here:
https://github.com/jbondc/TypeScript/commit/59919317bfeb1ea45717bdff19cb22e19c2296f6

Need to cleanup & pass tests but maybe will clarify what I'm trying to do and how it relates to enums.